JSON/Forget Perspective.json

```diff
@@ -57,5 +57,6 @@
   "Souvenir": { "Status": "NotACandidate" },
   "SteamID": "1831737244",
   "Symbol": "Fp",
+  "TutorialVideoUrl": "https://www.youtube.com/watch?v=X0li_huR0Eg&t",
```

Comment:
`&t` is redundant as I believe it is for specifying the start time of the video. Perhaps remove them for consistency with others.